mains/gdas.cc

```diff
@@ -52,6 +53,9 @@ int runApp(int argc, char** argv, const std::string traits, const std::string ap
   // Define a map from app names to lambda functions that create unique_ptr to Applications
   std::map<std::string, std::function<std::unique_ptr<oops::Application>()>> apps;
 
+  apps["convertstructuredgrid"] = []() {
```

Comment:
may want to make this and the other entry `converttostructuredgrid` just to be consistent with the App name

Reply:
okay will do!